### DISCUSSION TITLE
Chasseur

### DISCUSSION THREAD

**Discussion by david-roxygen** [secretcowlevel/DoomtrooperPublic#77] [Card Balance]
By comparing the old version and new version of chasseur, it is possible to observe some discrepancies. In old card game cybercurity mp was a small warrior and chasseur an intermediate warrior. Now cybercurity mp is better than chasseur as warrior. 

![Picture8](https://user-images.githubusercontent.com/88280663/130089980-4efb1049-70ec-488c-a5dd-2d85db7047c1.jpg)

Comparing ‘chasseur’ with warriors of other faction (like Infantry, sunset striker, clansman, young guard, venusian ranger, etc...) it is possible to observe that its stats are too low. As consequence, I suggest to improve the stats:

![Picture7](https://user-images.githubusercontent.com/88280663/130090110-12997f29-26d8-4aca-aa01-836abf61b670.jpg)

Current card:
Chasseur  (CYBERTRONIC)
CHASSEUR
When played AUGMENT 1 time
F: 1 S: 1 A: 4 V: 3

Suggested card:
Chasseur  (CYBERTRONIC)
CHASSEUR
When played AUGMENT 1 time
F: 3 S: 3 A: 4 V: 4

**Comment by Sol-au-fein** [comment]


**Reply by david-roxygen** [reply]
Could be nice. 

In my opinion depends on the destiny of coral beach. If coral beach maintain the current bonus (+2 to stat), it is better the version of chasseur with 4DP, and similar to old cards. If coral beach is changed (maybe as suggested in the other topic), this last 3DP version is better. 

the 4DP version with +2F Augment become similar to clansman, with +2S similar to young guard. 
Together to the current version of coral beach could become 7/5/6 or 5/7/6 warrior. That is decent.

**Comment by Sol-au-fein** [comment]
I deleted my previous comment, because i changed my mind. Looking at tabletop stats, chasseur have really high armor and equal fight/shoot attributes with elite forces from other factions. But they are also more expensive. Augments costs something. So 3/3/4/5 i guess.

`Chasseur are basic cyborgized combat troops. They have been equiped with subdermal armor and their bionic limbs make them many times stronger than normal human. Their semi robotic coolness gives them high leadership. Chasseur heroes are the best and the most competent of the Chasseurs. `

**Comment by david-roxygen** [comment]
I’m wondering, is the cybersecurity MP ability better than an AUGMENT? For me the answer is ‘yes’. As consequence, if cybersecurity MP is 3/3/4/3 warrior with a super ability, chasseur could have a little bit better stat.

Going up with DP cost could be reconsidered the original card. Converted in current game rule old chasseur was:

Old card:
Chasseur (CYBERTRONIC)
CHASSEUR
POWERIMMUNITY
F:5 S:2 A:4 V: 5

-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-•-
Old Powerimmunity could be changed with the augment. F stat decreased by one because there are no warrior with F or S 7 for 5 DP.

Hypothetical card:
Chasseur (CYBERTRONIC)
CHASSEUR
POWERIMMUNITY
F:4 S:2 A:4 V: 5
When played AUGMENT 1 time

Another warrior with similar stat is Desert Scorpion 

In general I think that chasseur will never be played because cybersecurity MP and VAC engineer are too better. The problem with chasseur isn't related to the card itself, it is related with the overall chasseur strategy. Probably chasseur and coral beach should be improved at the same time, and modyfied considering the two card at the same time. 

why someone should choose  chasseur when cybersecurity MP is so better. There is no strategy behind chasseur (all chasseur except vince). Mirrorman is good only to be converted in a bomb with brainwave bomb ;_;


XD probably CHASSEUR are the worst warriors of the whole game XD. Now also the cartel warriors are better  XD  XD  XD XD.

**Comment by 21-ric** [comment]
Chasseur (CYBERTRONIC)
CHASSEUR
When played AUGMENT 1 time.
Gains +1 to [F] , [S] , [A], and [V] for each other CHASSEUR in its squad.
F: 1 S: 1 A: 4 V: 3

**Reply by david-roxygen** [reply]
Could be nice. It answers to the lack of strategy behind all chasseur. I like this suggestion.

**Comment by david-roxygen** [comment]
to become a real valid alternative to cybersecurity MP also this version should be considered:

Suggested card:
Chasseur (CYBERTRONIC)
CHASSEUR
When played AUGMENT 1 time
F: 3 S: 3 A: 4 V: 3

![CHASSEUR_NEW](https://user-images.githubusercontent.com/88280663/137086366-d3a83b73-8d79-4b88-95fc-568eba187c21.png)

The value of the ability of cybersecurity MP can be considered equal to 1 augment in my opinion
